include/hyprutils/os/Process.hpp

```diff
@@ -25,7 +25,7 @@ namespace Hyprutils {
             const std::string& stdErr();
 
             // only populated when ran async
-            const pid_t pid();
+            const pid_t& pid();
```

Comment:
no api breaking

Reply:
that doesnt break the api, its just returning a reference instead of a copy

Reply:
```cpp
const std::string& Hyprutils::OS::CProcess::stdOut() {
    return out;
}

const std::string& Hyprutils::OS::CProcess::stdErr() {
    return err;
}
```

These are doing the same thing, its just to keep consistency

Reply:
breaks ABI for sure